bossroom.js

```diff
@@ -40,7 +48,15 @@ export const BossRoomBot = {
             yPosition: position != null ? position.y : null,
             zPosition: position != null ? position.z : null
         }
-        // console.log(`Using abilility ${ability} on targetId: ${targetId} at position: ${input.xPosition}, ${input.yPosition}, ${input.zPosition}`)
+        // console.log(`Using ability ${ability} on targetId: ${targetId} at position: ${input.xPosition}, ${input.yPosition}, ${input.zPosition}`)
         actionQueue.queue("PerformSkill", input)
+    },
+
+    moveTowards: (target, range, actionQueue) => {
```

Comment:
move towards and follow object don't mean the same thing to me, please rename this to followTarget or followObject

Reply:
Fixed!